Lmod/branches/7.0/z01-default_modules.sh

```diff
@@ -0,0 +1,13 @@
+if [ -z "$__Init_Default_Modules" -o -z "$LD_LIBRARY_PATH" ]; then
+   export __Init_Default_Modules=1;
+   if [ -z "$LMOD_SYSTEM_DEFAULT_MODULES" ];then
+     if [ -f "/usr/share/modulefiles/DefaultModules.lua" ];then
+       export LMOD_SYSTEM_DEFAULT_MODULES="DefaultModules"
+     else
+       export LMOD_SYSTEM_DEFAULT_MODULES="StdEnv"
+     fi
+   fi
```

Comment:
OK, this looks really good implementation allowing all different sites to implement their own customizations in any of 3 different possible ways... very well done!

Do we wish to merge this PR and address the other 2 issues in another branch? or, wait for this one?